### PR TITLE
Better fix for AIE2 lock count

### PR DESCRIPTION
**Pull request by stephenneuendorffer**

### CODE REVIEW

File: lib/Targets/AIETargetXAIEV1.cpp [hunk 1]
```diff
@@ -153,6 +152,7 @@ mlir::LogicalResult AIETranslateToXAIEV1(ModuleOp module, raw_ostream &output) {
       output << clear_range(col, row, 0x3F200, 0x3F3AC);
 
       // Release locks
+      int numLocks = target_model.getNumLocks(col, row);
       output << "for (int l=0; l<" << numLocks << "; l++)\n"
```
Comment (keryell) [inline lib/Targets/AIETargetXAIEV1.cpp:156]: While you are here, apply the coding style to the generated code
```suggestion
      int numLocks = target_model.getNumLocks(col, row);
      output << "for (int l = 0; l < " << numLocks << "; ++l)\n"
```

File: lib/Targets/AIETargetXAIEV2.cpp [hunk 2]
```diff
@@ -139,6 +138,7 @@ mlir::LogicalResult AIETranslateToXAIEV2(ModuleOp module, raw_ostream &output) {
       output << "XAie_CoreDisable(" << deviceInstRef << ", "
              << tileLocStr(col, row) << ");\n";
       // Release locks
+      int numLocks = target_model.getNumLocks(col, row);
       output << "for (int l=0; l<" << numLocks << "; l++)\n"
```
Comment (keryell) [inline lib/Targets/AIETargetXAIEV2.cpp:142]:
```suggestion
      int numLocks = target_model.getNumLocks(col, row);
      output << "for (int l = 0; l < " << numLocks << "; ++l)\n"
```

File: lib/Targets/AIETargetXAIEV1.cpp (hunk 1 above)
Comment (stephenneuendorffer) [reply lib/Targets/AIETargetXAIEV1.cpp:156]: strange that clang-format didn't catch it.

Comment (keryell) [reply lib/Targets/AIETargetXAIEV1.cpp:156]: Because it is hidden in a string. :-) Perhaps you need a kind ChatGPT-powered clang-format... ;-)

Comment (stephenneuendorffer) [reply lib/Targets/AIETargetXAIEV1.cpp:156]: Ah, right.

**Review by fifield — APPROVED** [review]
Makes sense to me since number of locks can vary by tile.

**Pull request merged**